capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/video_embed/video-embed.tpl.php

```diff
@@ -0,0 +1,31 @@
+<script type="text/javascript">
```

Comment:
Can we put the .javascript in a separate javascript file and include in the page trough drupal_add_js() in the c4m_features_homepage_video_embed_content_type_render() callback?

The javascript should be loaded in the footer.